release/models/rib/openconfig-rib-bgp-tables.yang

```diff
@@ -311,6 +321,639 @@ submodule openconfig-rib-bgp-tables {
+            type leafref {
+              path "../state/route-distinguisher";
+            }
+            description
+              "Reference to the rd list key";
+          }
+
+          container state {
+            description "Top level containern for l2vpn evpn RDs";
+            leaf route-distinguisher {
+              type oc-ni-types:route-distinguisher;
+              description
+                "The route distinguisher";
+            }
+          }
+          uses bgp-evpn-type-one-state;
+          uses bgp-evpn-type-two-state;
+          uses bgp-evpn-type-three-state;
+          uses bgp-evpn-type-four-state;
+          uses bgp-evpn-type-five-state;
+        }
+      }
+    }
+  }
+
+  grouping bgp-loc-rib-l2vpn-evpn-attr-refs {
+    description
+      "Definitions of common references to attribute sets for
+      multiple AFI-SAFIs for LOC-RIB tables";
+
+    leaf attr-index {
+      type leafref {
+        path "../../../../../../../../../../../../attr-sets/attr-set/" +
+          "state/index";
+      }
+      description
+        "Reference to the common attribute group for the
+        route";
+    }
+
+    leaf community-index {
+      type leafref {
+        path "../../../../../../../../../../../../communities/community/" +
+          "state/index";
+      }
+      description
+        "Reference to the community attribute for the route";
+    }
+
+    leaf ext-community-index {
+      type leafref {
+        path "../../../../../../../../../../../../ext-communities/" +
+          "ext-community/state/index";
+      }
+      description
+        "Reference to the extended community attribute for the
+        route";
+    }
+  }
+
+  grouping bgp-evpn-route-path-state {
+    description
+      "BGP l2vpn evpn route-type path state info grouping";
+
+    container paths {
+      description "List of BGP paths attributes for this route";
+      list path {
+        description "List of paths";
+        key "peer-ip peer-path-id";
+
+        leaf peer-ip {
+          type leafref {
+            path "../state/peer-ip";
+          }
+          description
+            "Reference to the peer-ip key";
+        }
+
+        leaf peer-path-id {
+          type leafref {
+            path "../state/peer-path-id";
+          }
+          description
+            "Reference to the peer-path-id key";
+        }
+
+        container state {
+          description "BGP path attributes for this route";
+
+          leaf peer-ip {
+            type oc-inet:ip-address;
+            description "Peer IP information";
+          }
+          leaf peer-path-id {
+            type uint32;
+            description "Peer path identifier";
+          }
+          leaf-list advertised-to-peer {
+            type oc-inet:ip-address;
+            description "List of peers to which this path is advertised";
+          }
+          leaf label {
+            type string;
+            description "Label info";
+          }
+          leaf label2 {
```

Comment:
is there any case where more than 2 labels are used?

Reply:
No